controllers/gatekeeper_controller_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"testing"
+
+	operatorv1alpha1 "github.com/font/gatekeeper-operator/api/v1alpha1"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/library-go/pkg/manifest"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestReplicas(t *testing.T) {
+	g := NewWithT(t)
+	auditReplicas := int64(4)
+	webhookReplicas := int64(7)
+	gatekeeper := &operatorv1alpha1.Gatekeeper{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test",
+			Namespace: "testns",
+		},
+	}
+	// test default audit replicas
+	auditManifest, err := getManifest(auditFile)
+	g.Expect(err).ToNot(HaveOccurred())
+	g.Expect(auditManifest).ToNot(BeNil())
+	testManifestReplicas(t, auditManifest, int64(1))
```

Comment:
I don't expect the default replica value (1 in this case) to change in upstream gatekeeper, but we should consider adding a constant var for these.

Reply:
@font done